agot-bg-game-server/data/baseGameData.json

```diff
@@ -7,32 +7,73 @@
     "kingsCourt": ["lannister", "stark", "martell", "baratheon", "tyrell", "greyjoy"]
   },
   "setups": {
-    "3": {
-      "houses": ["stark", "lannister", "baratheon"],
-      "blockedRegions": [
-        "pyke", "port-of-pyke", "highgarden", "oldtown", "port-of-oldtown", "sunspear",
-        "port-of-sunspear", "salt-shore", "yronwood", "starfall", "three-towers",
-        "dornish-marches", "princes-pass", "the-boneway", "storms-end", "port-of-storms-end"
-      ],
-      "removedUnits": ["port-of-lannisport"]
+    "base-game": {
+      "name": "2nd Edition Base Game (3 or 6 players)",
+      "playerSetups": [
+        {
+          "playerCount": 3,
+          "houses": ["stark", "lannister", "baratheon"],
+          "blockedRegions": [
+            "pyke", "port-of-pyke", "highgarden", "oldtown", "port-of-oldtown", "sunspear",
+            "port-of-sunspear", "salt-shore", "yronwood", "starfall", "three-towers",
+            "dornish-marches", "princes-pass", "the-boneway", "storms-end", "port-of-storms-end"
+          ],
+          "removedUnits": ["port-of-lannisport"]
+        },
+        {
+          "playerCount": 6,
+          "houses": ["stark", "greyjoy", "lannister", "baratheon", "tyrell", "martell"]
+        }
+      ]
     },
-    "4": {
-      "houses": ["stark", "greyjoy", "lannister", "baratheon"],
-      "blockedRegions": [
-        "highgarden", "oldtown", "port-of-oldtown", "sunspear", "port-of-sunspear", "salt-shore",
-        "yronwood", "starfall", "three-towers", "dornish-marches", "princes-pass", "the-boneway",
-        "storms-end", "port-of-storms-end"
+    "struggle-in-the-north": {
+      "name": "Struggle in the North (4 or 5 players)",
+      "playerSetups": [
+        {
+          "playerCount": 4,
+          "houses": ["stark", "greyjoy", "lannister", "baratheon"],
+          "blockedRegions": [
+            "highgarden", "oldtown", "port-of-oldtown", "sunspear", "port-of-sunspear", "salt-shore",
+            "yronwood", "starfall", "three-towers", "dornish-marches", "princes-pass", "the-boneway",
+            "storms-end", "port-of-storms-end"
+          ]
+        },
+        {
+          "playerCount": 5,
```

Comment:
I hesitated to encode the playerCount this way the first time. It makes the code to retrieve a specific game setup a bit more complex, but at least it is a true int and not a string of a number.

Reply:
Indeed. I tried it with the key value pair first but having a real array which then can be converted to a concrete type makes life so much easier...

Reply:
Ideally, it would be possible to use a number as a key JSON, that would make all operations on it easy. But yeah, let's keep your implementation.